querybook/server/lib/query_analysis/templating.py

```diff
@@ -205,18 +233,22 @@ def flatten_recursive_variables(
     flattened_variables = {}
     variables_dag = {}
 
+    global_variables = get_global_variables(jinja_env)
+
     for key, value in raw_variables.items():
         if not value:
             value = ""
 
-        variables_in_value = get_templated_variables_in_string(value, jinja_env)
+        variables_in_value = get_templated_variables_in_string(value)
```

Comment:
can you filter out the global functions in variables_in_value

Reply:
I changed variables_in_value to return global functions as well because templated variables with global functions in the value need to be flattened (otherwise the global function will be rendered the the template as the string of the function call - aka the string `'latest_partition(“table”, “dt”)'` will show up in the final rendered query)

Reply:
okay that makes sense!

Reply:
actually I don't think we need get_jinja_global_variables at all since `get_templated_variables_in_string(value, jinja_env)` would filter out the globals